### ISSUE TITLE
Swapping order of components gives same interaction parameters

### ISSUE THREAD

**Issue by ianhbell** [usnistgov/REFPROP-issues#265]
I seem to recall we had a workaround to this problem, but I don't recall what it was.  If I swap component order, I still get the same values for betaT, betaV, but they should be reciprocals, or I should get an error message, or some sort of feedback at least.  Currently I get the wrong values for one order with no information.  The lower example shows that this is handled properly in CoolProp when the component order is inverted.

```python
from ctREFPROP.ctREFPROP import REFPROPFunctionLibrary
root = 'D:/Program Files (x86)/REFPROP'
RP = REFPROPFunctionLibrary(root)
RP.SETPATHdll(root)

print(RP.RPVersion())
fld0 = 'R32'
fld1 = 'R1234YF'
for fldA, fldB in [(fld0, fld1), (fld1, fld0)]:
    setfl = RP.SETFLUIDSdll(fldA + '*' + fldB)
    ktv = RP.GETKTVdll(1, 2)
    r = RP.REFPROPdll('','','FIJMIX',0,0,0,1,2,[0.5,0.5])
    print(ktv)

import CoolProp.CoolProp as CP
for fluids in 'R32&R1234yf','R1234yf&R32':
    AS = CP.AbstractState('HEOS', fluids)
    for k in ['betaT','gammaT','betaV','gammaV','Fij']:
        print(k, AS.get_binary_interaction_double(0,1,k))
```
yielding
```
10.0072
GETKTVdlloutput(hmodij='KW4', fij=array('d', [1.00052, 0.948538, 0.993346, 1.02211, -0.277708, 0.0]), hFmix='HMX.BNC', hfij='BetaT   GammaT  BetaV   GammaV   Fij   not used', hbinp='R32/R1234yf\r\nR. Akasaka, FPE, 2013, DOI:10.1016/j.fluid.2013.07.057', hmxrul='KW4  Kunz and Wagner model for the methane-CO2 mixture')
GETKTVdlloutput(hmodij='KW4', fij=array('d', [1.00052, 0.948538, 0.993346, 1.02211, -0.277708, 0.0]), hFmix='HMX.BNC', hfij='BetaT   GammaT  BetaV   GammaV   Fij   not used', hbinp='R32/R1234yf\r\nR. Akasaka, FPE, 2013, DOI:10.1016/j.fluid.2013.07.057', hmxrul='KW4  Kunz and Wagner model for the methane-CO2 mixture')
betaT 1.00052
gammaT 0.948538
betaV 0.993346
gammaV 1.02211
Fij -0.277708
betaT 0.999480270259465
gammaT 0.948538
betaV 1.0066985723000847
gammaV 1.02211
Fij -0.277708
```

**Comment by ianhbell** [comment]
Seems the workaround is to query ``HMXORDER`` to figure out if the order is correct, but if you don't do that, you get the wrong values from either ``GETKTV`` or ``REFPROP`` without warning

**Comment by ianhbell** [comment]
Fixed in 10.0077
